### ISSUE TITLE
GameCube Port

### ISSUE THREAD

**Issue by TriVoxel** [xtreme8000/CavEX#44]
Is it possible this could be made to run on the GameCube? That would be SO COOL! I wouldn't even care if it had 1/4 the features and graphics tbh.

**Comment by xtreme8000** [comment]
At the moment it's using too much ram to run on a gamecube 😔
But it might work if the render distance is decreased.

**Comment by TriVoxel** [comment]
Yes, yes, yes! If you could implement a GC version with a very short render distance, that would be epic! Maybe you could also check for a GC Loader or an M.2 loader and if a high-speed storage device is available, offload some of the memory to a swap file? :-/

**Comment by TriVoxel** [comment]
Also, this is a whole other idea completely, but have you seen the Minecraft “Distant Horizons” mod works? It before a chunk is unloaded, it basically compresses the chunk into a low poly model, probably by averaging the blocks into larger blocks. But basically, when a chunk is really far away, the mod loads a super low detail version of it which increases visual fidelity and performance. These low poly LOD models don’t actually update until you actually reload the chunk, but who cares.

Maybe you could implement a similar system in your game to increase the draw distance by loading lower detail versions of the chunks in the distance.

I think you could basically just average every 2-4 blocks in the chunk and whichever block is most common, just turn it into 1 big block of that type to reduce polys and RAM usage… IDK, just an idea!
